test/broker_test/broker_test.go

```diff
@@ -687,10 +687,19 @@ var _ = test.DescribeTestsFor(test.TestCase{
 								WithJSON(postBrokerRequestWithLabels).
 								Expect().Status(http.StatusCreated).JSON().Object().Keys().Contains("id", "labels")
 						})
+
+						When("When creating labeled broker with a key containing a forbidden separator as substring", func() {
```

Comment:
Maybe "containing a separator not as a standalone word"? Feels strange saying "if you do something forbidden it should work well"